### PR TITLE
Test new issue templates

### PR DESCRIPTION
**Pull request by michaelstingl**

### CODE REVIEW

**Comment by michaelstingl** [comment]
Just found something fancy here:
https://github.com/owncloud/ios-app/community

Also:
https://github.blog/2018-05-02-issue-template-improvements/

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (jesmrec) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe, we can get rid of the `Desktop` section, and replace ir for a `Server` section with the details of the backend.

Comment (michaelstingl) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I was just playing with some GH stuff. No need for review yet.

**Comment by michaelstingl** [comment]
just experimenting with a GH feature – close for now…

**Pull request closed without merging**